webapp/src/components/DonationsDashboard/DonationsDashboard.js

```diff
@@ -0,0 +1,407 @@
+    marginBottom: 7,
+  },
+  draweTitleDesktop: {
+    color: "rgba(0, 0, 0, 0.87)",
+    fontFamily: "Roboto",
+    fontSize: "20px",
+    fontWeight: "bold",
+    fontStretch: "normal",
+    fontStyle: "normal",
+    lineHeight: "normal",
+    letterSpacing: "0.15px",
+    marginBottom: 7,
+  },
+  dividerTitle: {
+    marginBottom: 4,
+  },
+  boxBalance: {
+    display: 'flex',
+    flexDirection: 'column',
+    width: '100%',
+    alignItems: 'center',
+  },
+  balanceText: {
+    color: "#121212",
+    fontFamily: "Roboto",
+    fontSize: "14px",
+    fontWeight: "normal",
+    fontStretch: "normal",
+    fontStyle: "normal",
+    lineHeight: 1.14,
+    letterSpacing: "normal",
+    margin: 0,
+  },
+  heart: {
+    padding: 0,
+    width: '110px',
+    fontSize: 275,
+    animation: '$heartbeat 1.4s linear infinite',
+    [theme.breakpoints.up('md')]: {
+      width: '140px',
+    }
+  },
+  sectionTitle: {
+    color: "#000000",
+    fontFamily: "Roboto",
+    fontSize: "10px",
+    fontWeight: "500",
+    fontStretch: "normal",
+    fontStyle: "normal",
+    lineHeight: 1.6,
+    letterSpacing: "0.5px",
+    marginBottom: 0,
+  },
+  dividerSection: {
+    marginBottom: 4,
+    color: "#000000",
+  },
+  sectionText: {
+    color: "#000000",
+    fontFamily: "Roboto",
+    fontSize: "14px",
+    fontWeight: "normal",
+    fontStretch: "normal",
+    fontStyle: "normal",
+    lineHeight: 1.43,
+    letterSpacing: "0.25px",
+    marginBottom: 10,
+  },
+  boxQR: {
+    display: 'flex',
+    flexDirection: 'column',
+    width: '100%',
+    alignItems: 'center',
+    [theme.breakpoints.up('md')]: {
+      marginTop: 20,
+    }
+  },
+  accountText: {
+    color: "#000000",
+    fontFamily: "Roboto",
+    fontSize: "16px",
+    fontWeight: "normal",
+    fontStretch: "normal",
+    fontStyle: "normal",
+    lineHeight: 1.5,
+    letterSpacing: "0.15",
+    margin: 11,
+  },
+  inputText: {
+    width: "85%",
+  },
+  camaraButtonIcon: {
+    width: "10%",
+    marginTop: 5,
+    marginLeft: 10,
+  },
+  camaraIcon: {
+    color: "#000000",
+  },
+  sendTokenButton: {
+    backgroundColor: "#ba0d0d",
+    borderRadius: 50,
+    height: 40,
+    width: 140,
+    padding: 10,
+    color: "#ffffff",
+  },
+  boxButtonSendToken: {
+    display: 'flex',
+    flexDirection: 'column',
+    width: '100%',
+    alignItems: 'center',
+    marginTop: 30,
+    marginBottom: 20,
+  },
+  boxTexfield: {
+    width: '100%',
+    alignItems: 'center',
+    [theme.breakpoints.up('md')]: {
+      width: '70%',
+      margin: "auto",
+      marginTop: 20,
+    }
+  },
+  dialog: {
+    padding: 20
+  },
+  closeIcon: {
+    position: 'absolute',
+    zIndex: 1,
+    top: 10,
+    right: 10,
+    margin: '0',
+    height: "5vh",
+    '& svg': {
+      fontSize: 25,
+      color: "rgba(0, 0, 0, 0.6)"
+    }
+  },
+}));
+
+const EmptyHeartSVG = ({ balance, isDesktop }) => {
+  const classes = useStyles()
+  let textColor
+
+  if (isDesktop) {
```

Comment:
You could use a ternary operator here. Or remove the unnecessary curly braces here